### ISSUE TITLE
[TYPE 3] - Add "max_filesize" parameter to a workbook's `download` method

### ISSUE THREAD

**Issue by NFeruchBCG** [tableau/server-client-python#1500]
## Description
It would be helpful to be able to pass a `max_filesize` parameter to a workbook's download method that will return early if a workbook is too large 🙂

**Comment by jorwoods** [comment]
So you want the user to be able to set a maximum size, and if the workbook size exceeds this user defined value, don't download?

**Comment by NFeruchBCG** [comment]
Yes exactly - "only download the file its less than X units." Megabytes would probably make the most sense but open to whatever makes the most sense

**Comment by jorwoods** [comment]
Workbooks and datasources have a `size` attribute. Why can the user not just write their own `if` statement to check that prior to attempting the download?

**Comment by NFeruchBCG** [comment]
There's a problem with tableau's rest api, where the "Get Workbook" endpoint returns a constant `1` for all workbooks. Is this happening just for us?

**Comment by NFeruchBCG** [comment]
Actually sorry, for some workbooks - maybe workbooks less than 1 mb are being rounded up to 1?

**Comment by jorwoods** [comment]
Can you try checking how many you have of each?

```python
print("1mb workbooks", len(server.workbooks.filter(size="1")))
print(">1mb workbooks", len(server.workbooks.filter(size__gt="1")))
```
